src/views/LoanDetails/LoanDetails.js

```diff
@@ -164,22 +164,34 @@ class LoanDetails extends React.Component {
     const { stripes, resources } = this.props;
     const records = resources?.loanAccountsActions?.records ?? [];
     const total = records.reduce((acc, { amount }) => (acc + parseFloat(amount)), 0);
+    const suspendedAction = records.filter(a => a?.paymentStatus?.name === 'Suspended claim returned') || [];
+    const suspendedStatus = (suspendedAction.length > 0) ? 'Suspended' : '';
```

Comment:
`Suspended` needs to be localized, i.e. passed through `<FormattedMessage id="ui-users..." />`.

Reply:
@zburke  in this case a FormatMessage is called later